typings/index.d.ts

```diff
@@ -1,6 +1,6 @@
 /** @format */
 
-export { BaseScene, Stage } from './stage'
+export { BaseScene, WizardScene, Stage } from './stage'
```

Comment:
`WizardScene` is not exported from `telegraf.js`

Reply:
Fixed in new commit